glib/src/error.rs

```diff
@@ -82,6 +83,12 @@ impl Error {
                 .unwrap_or_else(|err| str::from_utf8(&bytes[..err.valid_up_to()]).unwrap())
         }
     }
+
+    pub fn into_raw(mut self) -> *mut ffi::GError {
+        let e = self.to_glib_none_mut().0;
+        mem::forget(self);
+        e
+    }
```

Comment:
These can all be updated to use the new function. Maybe I missed some elsewhere

```
gio/src/subclass/io_stream.rs
176:            *err = e.to_glib_none_mut().0;

gio/src/subclass/seekable.rs
213:            *err = e.to_glib_none_mut().0;
248:            *err = e.to_glib_none_mut().0;

gio/src/subclass/output_stream.rs
234:            *err = e.to_glib_none_mut().0;
258:            *err = e.to_glib_none_mut().0;
282:            *err = e.to_glib_none_mut().0;
314:            *err = e.to_glib_none_mut().0;

gio/src/subclass/input_stream.rs
193:            *err = e.to_glib_none_mut().0;
217:            *err = e.to_glib_none_mut().0;
251:            *err = e.to_glib_none_mut().0;
```

Reply:
We also have some in gtk4-rs, it would be very nice if you could update those as well otherwise an issue at least would be enough